app/assets/javascripts/intialize.js.erb

```diff
@@ -0,0 +1,7 @@
+$(function(){
+  $(document).foundation();
+  SC.initialize({
+    client_id: "<%= ENV['SOUNDCLOUD_APP_ID']%>",
+    redirect_uri: 'http://localhost:3000/callback'
```

Comment:
While you're here, it woudl be sweet to use a Rails initializer to change this based on the Rails environment (development, production, etc).  Then @torpeyp won't have to change the URL path while I'm pairing with him.

![Uploading truffle_shuffle.gif . . .]()